Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -367,4 +368,163 @@ public class Politician
         public string FirstName { get; init; }
         public string LastName { get; init; }
     }
+
+    public class NegationBoolIonSerializer : IonSerializer<bool>
+    {
+        public bool Deserialize(IIonReader reader)
+        {
+            return !reader.BoolValue();
+        }
+
+        public void Serialize(IIonWriter writer, bool item)
+        {
+            writer.WriteBool(!item);
+        }
+    }
+    
+    public class ZeroByteArrayIonSerializer : IonSerializer<byte[]>
+    {
+        public byte[] Deserialize(IIonReader reader)
+        {
+            return new byte[reader.GetLobByteSize()];
+        }
+
+        public void Serialize(IIonWriter writer, byte[] item)
+        {
+            writer.WriteBlob(new byte[item.Length]);
+        }
+    }
+    
+    public class UpperCaseStringIonSerializer : IonSerializer<string>
+        {
+        public string Deserialize(IIonReader reader)
+        {
+            return reader.StringValue().ToUpper();
+        }
+
+        public void Serialize(IIonWriter writer, string item)
+        {
+            writer.WriteString(item.ToUpper());
+        }
+    }
+
+    public class NegativeIntIonSerializer : IonSerializer<int>
+    {
+        public int Deserialize(IIonReader reader)
+        {
+            return -reader.IntValue();
+        }
+
+        public void Serialize(IIonWriter writer, int item)
+        {
+            writer.WriteInt(-item);
+        }
+    }
+
+    public class NegativeLongIonSerializer : IonSerializer<long>
+    {
+        public long Deserialize(IIonReader reader)
+        {
+            return -reader.LongValue();
+        }
+
+        public void Serialize(IIonWriter writer, long item)
+        {
+            writer.WriteInt(-item);
```

Comment:
Should be WriteLong?

Reply:
There is no `WriteLong()` in Ion Writer.
Also it appears `WriteInt()` is meant to handle longs because it takes in a long.